library/fiber/raise.rb

```diff
@@ -0,0 +1,91 @@
+require_relative '../../spec_helper'
+
+require 'fiber'
+
+ruby_version_is "2.7" do
+  describe "Fiber#raise" do
+
+    it 'raise RuntimeError by default' do
```

Comment:
The spec descriptions should form a sentence, so typically the first word will be a 3rd person present verb like `raises/accepts/does not accept`, etc.